OpenSim/Simulation/InverseKinematicsSolver.h

```diff
@@ -140,16 +128,23 @@ class OSIMSIMULATION_API InverseKinematicsSolver: public AssemblySolver
         assembly solver. Subclasses can override to include other goals  
         such as point of interest matching (Marker tracking). This method is
         automatically called by assemble. */
-    virtual void setupGoals(SimTK::State &s) override;
+    void setupGoals(SimTK::State &s) override;
     /** Internal method to update the time, reference values and/or their 
         weights that define the goals, based on the passed in state */
-    virtual void updateGoals(const SimTK::State &s) override;
+    void updateGoals(const SimTK::State &s) override;
 
 private:
+    // The marker reference values and weightings
+    MarkersReference &_markersReference;
+
+    // Markers collectively form a single assembly condition for the SimTK::Assembler
+    // and the memory is managed by the Assembler
+    SimTK::ReferencePtr<SimTK::Markers> _markerAssemblyCondition;
+
     // Non-accessible cache of the marker values to be matched at a given state
     SimTK::Array_<SimTK::Vec3> _markerValues;
 
-
+    std::unique_ptr<SimTK::OrientationSensors> _orientationAssemblyCondition;
```

Comment:
As far as I can tell, this is unused. Also, I didn't think you could make a `std::unique_ptr` member variable because I though `std::unique_ptr` is not copyable. Anywho, how about commenting this out for now until we support orientation sensors?